cs-xml/src/main/java/io/cloudslang/content/xml/entities/inputs/ApplyXslTransformationInputs.java

```diff
@@ -0,0 +1,70 @@
+package io.cloudslang.content.xml.entities.inputs;
+
+import io.cloudslang.content.utils.StringUtilities;
+import io.cloudslang.content.xml.utils.Constants;
+
+/**
+ * Created by moldovas on 08/07/2016.
+ */
+public class ApplyXslTransformationInputs {
+    private String xmlDocument;
+    private String xslTemplate;
+    private String outputFile;
+    private String parsingFeatures;
+
+    private ApplyXslTransformationInputs(ApplyXslTransformationBuilder builder) {
+        this.xmlDocument = builder.xmlDocument;
+        this.xslTemplate = builder.xslTemplate;
+        this.outputFile = builder.outputFile;
+        this.parsingFeatures = builder.parsingFeatures;
+    }
+
+    public String getXmlDocument() {
+        return xmlDocument;
+    }
+
+    public String getXslTemplate() {
+        return xslTemplate;
+    }
+
+    public String getOutputFile() {
+        return outputFile;
+    }
+
+    public String getParsingFeatures() {
+        return parsingFeatures;
+    }
+
+    public static class ApplyXslTransformationBuilder {
```

Comment:
I suggest renaming this class simply to Builder. The correct name would be ApplyXslTransformation**Inputs**Builder, but since it's a nested builder class, it's clear that it builds a ApplyXslTransformationInputs object. Renaming this to Builder is less verbose. But this is not a blocker.